src/server.js

```diff
@@ -325,11 +326,6 @@ function start(opts) {
               Math.floor(centerPx[1] / 256) + '.' + data_.format;
         }
 
-        var query = req.query.key ? ('?key=' + req.query.key) : '';
-        data_.wmts_link = 'http://wmts.maptiler.com/' +
-          base64url('http://' + req.headers.host +
```

Comment:
Was this the only reference to the `base64url` package? I guess we could remove the dependency completely...